### PR TITLE
Jb54049

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Just the last commit belongs to this PR

File: share/qtcreator/sfdk/modules/50-testing-mb2.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
         },
         {
             "name": "check",
-            "trSynopsis": "--list-suites\n[-l|--levels [+|-]<level>...] [-s|--suites [+|-]<suite>...] [<artefac>]...",
+            "trSynopsis": "--list-suites\n[-l|--levels [+|-]<level>...] [-s|--suites [+|-]<suite>...] [--] [<artefac>]...",
```
Comment (vigejolla) [inline share/qtcreator/sfdk/modules/50-testing-mb2.json:35]: Could you add the missing 't' to the word 'artefact' while at it? Otherwise, LGTM

**Review by vigejolla — APPROVED** [review]

File: share/qtcreator/sfdk/modules/50-testing-mb2.json (hunk 1 above)
Comment (martyone) [reply share/qtcreator/sfdk/modules/50-testing-mb2.json:35]: The commit is on the bottom, I will push another commit after merging all the PRs.

Comment (martyone) [reply share/qtcreator/sfdk/modules/50-testing-mb2.json:35]: Added to PR #444

**Pull request merged**